### PR TITLE
OpenStack: troubleshooting: add broken machine deletion section

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: docs/user/openstack/troubleshooting.md [hunk 1]
```diff
@@ -15,6 +15,18 @@ OpenStack CLI tools should be installed, then:
 
 Destroying the cluster has been noticed to [sometimes fail](https://github.com/openshift/installer/issues/1985). We are working on patching this, but in a mean time the workaround is to simply restart the destroying process of the cluster.
 
+## Machine has ERROR state
+
+This could be because the machine's instance was accidentally destroyed and the cluster API provider cannot recreate it.
+
+You can check the status of machines with the help of the command `oc get machines -n openshift-machine-api`.
+
+If a machine has the error state, you should delete it manually with `oc delete machine -n openshift-machine-api <machine_name>` command. The operation can take up to 5 minutes, during which time the machine will be gracefully deleted and all resources returned to the pool.
```
Comment (abhinavdahiya) [inline docs/user/openstack/troubleshooting.md]: Hmm, what if the users deletes the control-plane machine... if you delete that object, no new machine object will be created as it is not parented by a machineset.

Comment (Fedosin) [reply docs/user/openstack/troubleshooting.md]: added this section too

File: docs/user/openstack/troubleshooting.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ OpenStack CLI tools should be installed, then:
 
 Destroying the cluster has been noticed to [sometimes fail](https://github.com/openshift/installer/issues/1985). We are working on patching this, but in a mean time the workaround is to simply restart the destroying process of the cluster.
 
+## Machine has ERROR state
+
+This could be because the machine's instance was accidentally destroyed and the cluster API provider cannot recreate it.
+
+You can check the status of machines with the help of the command `oc get machines -n openshift-machine-api`.
```
Comment (iamemilio) [inline docs/user/openstack/troubleshooting.md]: Keep code snippets consistent with the rest of the doc

````suggestion

You can check the status of machines with the command 

```sh
oc get machines -n openshift-machine-api
```
````

File: docs/user/openstack/troubleshooting.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ OpenStack CLI tools should be installed, then:
...
+
+If the broken machine is a master then follow the instructions in the [disaster recovery documentation](https://docs.openshift.com/container-platform/4.1/disaster_recovery/scenario-1-infra-recovery.html).
+
+For workers, you should delete the machine manually with `oc delete machine -n openshift-machine-api <machine_name>` command. The operation can take up to 5 minutes, during which time the machine will be gracefully removed and all its resources returned to the pool.
```
Comment (iamemilio) [inline docs/user/openstack/troubleshooting.md]: ^

File: docs/user/openstack/troubleshooting.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ OpenStack CLI tools should be installed, then:
...
+
+A new worker machine for the cluster will soon be created automatically by the [operator](https://github.com/openshift/machine-api-operator).
```
Comment (iamemilio) [inline docs/user/openstack/troubleshooting.md]:
```suggestion
A new worker machine for the cluster will soon be created automatically by the [machine-api-operator](https://github.com/openshift/machine-api-operator).
```

File: docs/user/openstack/troubleshooting.md [hunk 5]
```diff
@@ -15,6 +15,20 @@ OpenStack CLI tools should be installed, then:
...
+
+**NOTE**: in future versions of OpenShift all broken machines will be automatically deleted and recovered by the operator.
```
Comment (iamemilio) [inline docs/user/openstack/troubleshooting.md]: which operator?

**Pull request merged**